contracts/Pool/PoolRegistry.sol

```diff
@@ -345,13 +350,15 @@ contract PoolRegistry is Ownable2StepUpgradeable, AccessControlledV8, PoolRegist
     }
 
     /**
-     * @notice Update metadata of an existing pool.
+     * @notice Update metadata of an existing pool
+     * @param comptroller Pool's Comptroller
+     * @param metadata_ New pool metadata
      */
-    function updatePoolMetadata(address comptroller, VenusPoolMetaData memory _metadata) external {
+    function updatePoolMetadata(address comptroller, VenusPoolMetaData memory metadata_) external {
```

Comment:
I think we can use `calldata` instead of `memory` as metadata_ is never modified in this function

Reply:
Fixed in dcbc9f9